src/main/java/io/jenkins/plugins/designlibrary/Colors.java

```diff
@@ -12,14 +13,124 @@ public String getIconFileName() {
         return "symbol-colors";
     }
 
-    public List<String> getColors() {
-        // Uncomment these once these colors are in core
-        // return Arrays.asList("red", "green", "orange", "yellow", "blue", "indigo", "violet");
-        return Arrays.asList("red", "green", "orange");
+    public List<Semantic> getSemantics() {
+        return Arrays.asList(
+                new Semantic("Success", "Use for success messages", "success-color"),
+                new Semantic("Warning", "Use for warning messages", "warning-color"),
+                new Semantic("Error", "Use for error messages", "error-color"),
```

Comment:
*messages* seems a bit too restrictive for me, in my plugins I will show line or bar charts, or an icon with this semantic colors as well. (I don't know an English word that makes a broader range for the usage clear, maybe you have an idea? Or use something like *messages, states, items*)

Reply:
Agreed 100%.  'States' is the first thing that came to my mind in response to your comment, I've updated it the wording :) Thanks!